### PR TITLE
ESLint Part 1

### PR DESCRIPTION
**Pull request by tarrball**

### CODE REVIEW

File: src/DashboardUI/src/data-contracts/index.ts [hunk 1]
```diff
@@ -4,7 +4,6 @@ export * from './WaterSourceInfoListItem';
 export * from './SiteLocation';
 export * from './MapBoundSettings';
 export * from './BeneficialUseListItem';
-export * from './feature';
```
Comment (tarrball) [inline src/DashboardUI/src/data-contracts/index.ts:7]: note this was an unused and empty interface

File: src/DashboardUI/package.json [hunk 2]
```diff
@@ -60,13 +60,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
```
Comment (zlannin) [inline src/DashboardUI/package.json:69]: Why don't we want this for react-app and react-app/jest?

Comment (tarrball) [reply src/DashboardUI/package.json:69]: Config is just moved to a different file (eslint.config.mjs), out of package.json

**Comment by zlannin** [comment]
Can we close this?

**Pull request closed without merging**